File: src/main/java/com/avs/filmoteca/utils/RequestUtils.java
package com.avs.filmoteca.utils;

import javax.servlet.http.HttpServletRequest;

import com.avs.filmoteca.data.db.DataBase.Environment;

public class RequestUtils {

	public static Environment getEnvironment(HttpServletRequest request) {
		String pathInfo = request.getServletPath(); // /{value}/test
		if (pathInfo == null)
			return Environment.PRO;
		String[] pathParts = pathInfo.split("/");
		if (pathParts.length > 0)
			return Environment.findByPath(pathParts[1]);
		return Environment.PRO;
	}

}

File: src/main/kotlin/filmoteca/utils/RequestUtils.kt
package filmoteca.utils

import filmoteca.data.db.DataBase.Environment

object RequestUtils {

    fun getEnvironment(environment: String?): Environment {
        return when (environment) {
            "pro" -> Environment.PRO
            "dev" -> Environment.DEV
            else -> Environment.DEV
        }
    }

}
